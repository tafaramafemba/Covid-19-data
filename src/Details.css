@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;1,100;1,200;1,300;1,400;1,500;1,600&family=Roboto+Slab:wght@900&display=swap');

.all-details {
  display: flex;
  flex-direction: column;
  font-family: 'Lato', sans-serif;
  color: #fff;
}

.list {
  display: flex;
  flex-direction: column;
  font-size: 20px;
  row-gap: 50px;
}

.country-title {
  font-family: 'Lato', sans-serif;
  font-size: 30px;
  color: #fff;
}

.title {
  background-color: #4369b2;
  background-image: url('../public/covid.png');
  background-repeat: no-repeat;
  background-position: 200px 60px;
  background-size: contain;
  padding-bottom: 100px;
  border-top: 1px solid #fff;
}

.det-itm {
  font-weight: bold;
}
