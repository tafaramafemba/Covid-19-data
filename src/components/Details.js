function Details(){
  return(
    <h1>Hie</h1>
  )
}

export default Details;