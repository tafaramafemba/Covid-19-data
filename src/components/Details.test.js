import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Details from './Details';

describe(' Details Component Rendering ', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
      <Details />
    </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});