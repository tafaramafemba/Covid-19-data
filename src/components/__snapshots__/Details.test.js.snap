// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Details Component Rendering  renders correctly 1`] = `
<div>
  <div
    className="title"
  >
    <div
      className="details-item"
    />
  </div>
</div>
`;
